### ISSUE TITLE
Update Swedish translations

### ISSUE THREAD

**Issue by curmalin** [squidfunk/mkdocs-material#5164]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "sv",
  "action.edit": "Redigera sidan",
  "action.skip": "Gå till innehållet",
  "action.view": "Visa källkoden för denna sida",
  "announce.dismiss": "Visa inte igen",
  "blog.archive": "Arkivera",
  "blog.categories": "Kategorier",
  "blog.categories.in": "i",
  "blog.continue": "Fortsätt läsa",
  "blog.draft": "Utkast",
  "blog.index": "Tillbaka till index",
  "blog.meta": "Metadata",
  "blog.references": "Relaterade länkar",
  "clipboard.copy": "Kopiera till urklipp",
  "clipboard.copied": "Kopierat till urklipp",
  "consent.accept": "Acceptera",
  "consent.manage": "Hantera inställningar",
  "consent.reject": "Acceptera inte",
  "footer": "Sidfot",
  "footer.next": "Nästa",
  "footer.previous": "Föregående",
  "header": "Sidhuvud",
  "meta.comments": "Kommentarer",
  "meta.source": "Källa",
  "nav": "Navigation",
  "readtime.one": "1 min lästid",
  "readtime.other": "# min lästid",
  "rss.created": "RSS-flöde",
  "rss.updated": "RSS-flöde av uppdaterat innehåll",
  "search": "Sök",
  "search.placeholder": "Sök",
  "search.share": "Dela",
  "search.reset": "Rensa",
  "search.result.initializer": "Initialiserar sök",
  "search.result.placeholder": "Skriv sökord",
  "search.result.none": "Inga sökresultat",
  "search.result.one": "1 sökresultat",
  "search.result.other": "# sökresultat",
  "search.result.more.one": "1 till på denna sida",
  "search.result.more.other": "# till på denna sida",
  "search.result.term.missing": "Saknas",
  "select.language": "Välj språk",
  "select.version": "Välj version",
  "source": "Gå till datakatalog",
  "source.file.contributors": "Författare",
  "source.file.date.created": "Skapad",
  "source.file.date.updated": "Senast uppdaterad",
  "tabs": "Flikar",
  "toc": "Innehållsförteckning",
  "top": "Tillbaka till toppen"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in e11b70874.

**Comment by squidfunk** [comment]
Released as part of 9.1.2.
